experiment/simulation/css/main.css

```diff
@@ -1 +1,457 @@
+}
+
+.menu-options {
+  list-style: none;
+  padding: 10px 0;
+  z-index: 1;
+}
+
+.menu-option {
+  font-weight: 500;
+  z-index: 1;
+  font-size: 14px;
+  padding: 10px 40px 10px 20px;
+}
+
+.menu-option:hover {
+  background: rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+}
+
+.component-button{
+  width: 80px;
+  height: 80px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  margin: 0.5rem 0.5rem;
+  font-weight: bold;
+  font-size: 0.8rem;
+  text-align: center;
+  color: black;
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+@media screen and (max-width: 600px) {
+  .component-button {
+    width: 60px;
+    height: 60px;
+    background-size: 100%;
+  }
+}
+
+.component-button::after {
+  display: block;
+  padding-bottom: 100%;
+}
+
+.component-button:hover {
+  border: 0.2rem solid rgb(157, 210, 248);
+  cursor: pointer;
+}
+
+
+
+.success-message {
+  font-size: 1.2rem;
+  color: rgb(31, 170, 31);
+  font-weight: bold;
+}
+
+.failure-message {
+  font-size: 1.2rem;
+  color: rgb(228, 23, 23);
+  font-weight: bold;
+}
+
+.failure-input {
+  font-size: 0.8rem;
+  color: rgb(228, 23, 23);
+  font-weight: bold;
+}
+
+.modal {
+  position: fixed;
+  z-index: 10;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  opacity: 0;
+  visibility: hidden;
+  transform: scale(1.1);
+  transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.modal-content {
+  position: absolute;
+  z-index: 10;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: white;
+  padding: 1rem 1.5rem;
+  width: 24rem;
+  border-radius: 0.5rem;
+}
+
+.close-button {
+  z-index: 10;
+  float: right;
+  width: 1.5rem;
+  line-height: 1.5rem;
+  text-align: center;
+  cursor: pointer;
+  border-radius: 0.25rem;
+  background-color: lightgray;
+}
+
+.close-button:hover {
+  background-color: darkgray;
+}
+
+.show-modal {
+  opacity: 1;
+  visibility: visible;
+  transform: scale(1.0);
+  transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
+}
+
+/* for dropping instruction box  */
+
+.instructions-box {
+  max-height: 70px;
+  overflow: hidden;
+  transition: all .5s ease;
+}
+
+.instructions-box.expand {
+  max-height: 400px;
+}
+
+.v-instruction-container {
+  border:.2rem solid #97d23d;
+  justify-content: flex-start;
+}
+
+.highlight {
+  border:3px solid red;
+  box-shadow: 0 0 10px #719ECE;
+}
+
+#prompt-form {
+  display: inline-block;
+  padding: 5px 5px 5px 5px;
+  width: 350px;
+  border: 1px solid black;
+  background: rgb(200, 234, 243);
+  vertical-align: middle;
+  align-items: center;
+}
+
+#prompt-form-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 9999;
+  display: none;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+}
+
+#prompt-form-container:before {
+  display: inline-block;
+  height: 100%;
+  content: '';
```

Comment:
Do we still need content even if it empty?

Reply:
removed that instead increased the top percentage